packages/contracts/test/plugins/governance/multisig/multisig.ts

```diff
@@ -73,6 +73,28 @@ export async function approveWithSigners(
   await Promise.all(promises);
 }
 
+async function advanceTime(time: number) {
+  if (hre.network.config.zksync) {
+    time = time / 1000;
+  }
+  await ethers.provider.send('evm_increaseTime', [time]);
+  await ethers.provider.send('evm_mine', []);
+}
+
+function expectedBlockNumber(blockNumber: number) {
+  if (hre.network.config.zksync) {
+    return blockNumber - 2;
+  }
+  return blockNumber - 1;
```

Comment:
This is not correct. The fact that tests sometimes contain `blockNumber - 1` is that we have to look at the previous block.
But this doesn't mean that this is the case for all expected block numbers

Reply:
The hardhat matchers and evm modes for zksync work differently. I am not sure what you're asking.